source/main/modules/window-manager/index.ts

```diff
@@ -116,6 +116,10 @@ export default class WindowManager extends EventEmitter {
       .then(() => {
         global.log.info('[Window Manager] Window Manager booted. Opening main window.')
         this.showMainWindow()
+        if (process.argv.includes('--tray')) {
+          global.tray.add()
```

Comment:
This looks as if it might break, since the Tray provider relies on the global configuration, and if the tray has been created and THEN the user activates the corresponding setting, it might break something. I would rather argue that we use the global config setting instead of circumventing that. (I.e. set it if it's not set, else only hide the main window)

Additionally, if we add a new CLI option, it must be documented in the README file.

Reply:
Also if the config `system.leaveAppRunning` is false, --tray won't do anything. The user would expect --tray to do something.

Reply:
That is true!
Should we introdude a hidden state 'system.leaveAppRunningOverride' as to not modify the original setting? 

That way, we have an indepdendend way to enable this behaviour, and this would also allow to add a UI-toggle for starting in tray.

Reply:
A second, independent setting would be great. We can also conditionally enable/disable the `Show in tray` depending on the `start minimized` (or vice versa), so that the user sees that if you start minimized this requires the tray to be set.